polkadot/runtime/common/src/claims.rs

```diff
@@ -171,7 +176,14 @@ pub mod pallet {
 	pub trait Config: frame_system::Config {
 		/// The overarching event type.
 		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
-		type VestingSchedule: VestingSchedule<Self::AccountId, Moment = BlockNumberFor<Self>>;
+		type VestingSchedule: VestingSchedule<
+			Self::AccountId,
+			Moment = BlockNumberFor<Self>,
+			Fungible = Self::Fungible,
+		>;
+		type Fungible: fungible::Inspect<Self::AccountId, Balance = Self::Balance>
```

Comment:
```suggestion
		type Currency: fungible::Inspect<Self::AccountId, Balance = Self::Balance>
```

Reply:
Or `type NativeToken`, as suggested in https://github.com/paritytech/polkadot-sdk/issues/226#issuecomment-1691692395